query/src/servers/http/v1/query/execute_state.rs

```diff
@@ -107,6 +107,11 @@ impl Executor {
             if kill {
                 r.session.force_kill_query();
             }
+            // Write Finish to query log table.
+            r.interpreter
+                .finish()
+                .await
+                .map_err(|e| log::warn!("interpreter.finish error: {:?}", e));
```

Comment:
sorry, map_err should be unwrap_or_else,  my local test did not finish

Reply:
Fixed